### ISSUE TITLE
Error Loading World

### ISSUE THREAD

**Issue by Jaha123** [Yogu/itemfinder#5]
Invalid NBT  implementation state: Type com.evilco.mc.nbt.tag.TagCompound in (com.evilco.mc.nbt.stream.NbtlnputStream) has no valid constructor: null

I don't know if your still updating this app but if you could tell me what im doing wrong that would be great. :3

**Comment by Yogu** [comment]
Oh sorry, I've not used or developed itemfinder for a long time, and I probably won't have time to look at this...

**Comment by Jaha123** [comment]
Yeah I can understand that. I gave up anyway :(. But if you do ever get
round to fixing it don’t forget to message me

**Comment by MakesWhat** [comment]
I would also love to see this updated. It's the only thing of its kind I've seen after scouring the internet for a few minutes for one.
